db/migrate/20210802093531_add_parent_id_and_settings_to_accounts.rb

```diff
@@ -0,0 +1,7 @@
+class AddParentIdAndSettingsToAccounts < ActiveRecord::Migration[5.2]
+  def change
+    add_reference :accounts, :parent, foreign_key: { to_table: :accounts }
+    add_column :accounts, :settings, :jsonb, default: {}
```

Comment:
this migration will need to be adjusted as the settings stuff is already in

Reply:
Ok noted thanks.